mne/channels/montage.py

```diff
@@ -348,6 +348,7 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         selection = selection[sel]
     else:
         ch_names_ = list(ch_names_)
+    ch_names_ = [str(ch) for ch in ch_names_]  # convert names to str
```

Comment:
Agreed, we shouldn't end up with bytes, it means some earlier function needs to be fixed.

FYI with bytes we usually use `b.decode(utf-8)` instead of `str` because it's UTF-8 on Python 2.